package.json

```diff
@@ -150,8 +150,8 @@
     "babel-polyfill": "^6.26.0",
     "chai": "^4.2.0",
     "eslint": "^5.5.0",
-    "react": "~16.7.0",
-    "react-dom": "~16.6.0",
+    "react": "^16.7.0",
```

Comment:
Is there anything from `16.7` that ui-circulation depends on?  Looks like `stripes` and `platform-core` are requesting `~16.6` which ui-circulation will complain about during a platform install since its less than the minor version requested here.

Reply:
@ttamj that's what started causing issues for me. I'm not sure if `16.7` is needed here.

Reply:
@ttamj It is not necessary. I just got a problem with duplication of React and I have fixed it in such a way and probably forgot to delete this dependency. So just feel free to revert it to `~16.6`